recipes/mss/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
```

Comment:
@ReimarBauer what we need here is to user a tarball instead of the git repository to avoid a huge download. Here is how this section should look like:

```yaml
source:
  fn: mss-{{ version }}.tar.gz
  url: http://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
  sha256: 0529bf9e41b878de4b82278c52469cd88d95531e4e4296bac71f7f44f6d3f93b
```

You can copy and paste that over your `source` section above.